pkg/cluster/internal/create/actions/config/config.go

```diff
@@ -201,7 +201,7 @@ func getKubeadmConfig(cfg *config.Cluster, data kubeadm.ConfigData, node nodes.N
 	// configure the right protocol addresses
 	if cfg.Networking.IPFamily == "ipv6" {
 		if nodeAddressIPv6 == "" {
-			return "", errors.Errorf("failed to get IPV6 address; is the docker daemon configured to use IPV6 correctly?")
+			return "", errors.Errorf("failed to get IPV6 address; is the container provider (docker,podman) configured to use IPV6 correctly?")
```

Comment:
TODO: we can insert the name of the one we're using here, there's a .String() on provider

Reply:
(we want to avoid useless detail / sending people looking at the wrong thing)

Reply:
https://github.com/kubernetes-sigs/kind/pull/1960